src/main/java/racingcar/domain/Winners.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.domain;
+
+import racingcar.domain.car.Cars;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Winners {
+    public static final String AND = ", ";
+    private final List<String> winners;
+
+    public Winners(Cars cars) {
+        int maxLocation = findMaxLocation(cars);
+        List<String> winners = findWinners(cars, maxLocation);
+        this.winners = winners;
+    }
+
+    public int findMaxLocation(Cars cars) {
+        List<Integer> locationOfCars = cars.getCars().stream()
+                .map(car -> car.getLocation())
```

Comment:
```suggestion
                .map(Car::getLocation)
```

위와 같은 코드를 메서드 레퍼런스라고 부르는데요.
자바8 부터는 메서드 레퍼런스를 제공하고 있으니 활용하여 가독성을 올릴 수 있습니다!

Reply:
한번 활용해봤습니다 :)